packages/ubuntu_wizard/test/validated_form_field_test.dart

```diff
@@ -200,4 +200,22 @@ void main() {
     expect(find.text('equal'), findsOneWidget);
     expect(find.text('not equal'), findsNothing);
   });
+
+  testWidgets('focus', (tester) async {
+    final focusNode = FocusNode();
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: Material(
+          child: ValidatedFormField(autofocus: true, focusNode: focusNode),
```

Comment:
One minor question: would there be an interesting test case in which we could explore `autofocus: false`? Just wondering the value of testing `autofocus: true` when the purpose of the PR is to ensure fine-grained control over the focus. Or, instead, should we pay more attention to the fact that the API exposes the `focusNode` as the commit message states and checked at the following expectation:

```dart
expect(
      tester.widget<TextField>(find.byType(TextField)).focusNode,
      equals(focusNode),
    );
```

Reply:
Thanks for nudging me to add more tests! :smile: